spec/cleaner_spec.rb

```diff
@@ -276,6 +276,134 @@ def id
       expect(cleaner.clean_object(params)).to eq({ request: { params: { foo: { bar: 'baz' } } } })
     end
 
+    describe "redacted keys" do
+      it "redacts exact string match" do
+        object = { events: { metaData: { foo: 'bar' } } }
+
+        configuration = Bugsnag::Configuration.new
+        configuration.redacted_keys = Set['foo']
+
+        cleaner = Bugsnag::Cleaner.new(configuration)
+        expect(cleaner.clean_object(object)).to eq({ events: { metaData: { foo: '[FILTERED]' } } })
```

Comment:
This is probably going to complicate the change a lot, but the notifier spec string is "[REDACTED]" (not "FILTERED").

Reply:
Yeah I figured that wasn't particularly important — it can changed to say "[REDACTED]" when `meta_data_filters` are removed